splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/SpliceSystemProcedures.java

```diff
@@ -379,14 +379,40 @@ public SpliceSystemProcedures(DataDictionary dictionary) {
                     Procedure collectNonMergedSampleStatsForTable = Procedure.newBuilder().name("COLLECT_NONMERGED_TABLE_SAMPLE_STATISTICS")
                             .numOutputParams(0)
                             .numResultSets(1)
-                            .varchar("schema",128)
+                            .varchar("schema",1024)
                             .varchar("table",1024)
                             .arg("samplePercentage", DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.DOUBLE).getCatalogType())
                             .arg("staleOnly", DataTypeDescriptor.getBuiltInDataTypeDescriptor(Types.BOOLEAN).getCatalogType())
                             .ownerClass(StatisticsAdmin.class.getCanonicalName())
                             .build();
                     procedures.add(collectNonMergedSampleStatsForTable);
 
+                    Procedure fakeStatsForTable = Procedure.newBuilder().name("FAKE_TABLE_STATISTICS")
+                            .numOutputParams(0)
+                            .numResultSets(1)
+                            .modifiesSql()
+                            .varchar("schema",1024)
+                            .varchar("table",1024)
```

Comment:
table and schema names are catalog type. See this example:
Procedure importWithBadRecords = Procedure.newBuilder().name("IMPORT_DATA")
                            .numOutputParams(0).numResultSets(1).ownerClass(HdfsImport.class.getCanonicalName())
                            **.catalog("schemaName")
                            .catalog("tableName")**
                            ...

Reply:
Thanks @jyuanca I've made the change as requested.